website/views/shared/modals/settings.jade

```diff
@@ -66,6 +66,11 @@ script(type='text/ng-template', id='modals/delete.html')
     .row
       .col-md-6
         input.form-control(type='password', ng-model='_deleteAccount')
+    br
+    .row
+      .col-xs-12.form-group
+        label(for='feedback')=env.t('feedback')
+        textarea#feedback.form-control(ng-model='feedback')
```

Comment:
Haha yeah, its been a rough transition back into web dev. Thanks for pointing that out and taking the time to make this easier on me, I really appreciate it.